src/redux/actions/experimentSettings/loadProcessingSettings.js

```diff
@@ -34,10 +37,19 @@ const loadProcessingSettings = (experimentId) => async (dispatch, getState) => {
 
     throw new Error('HTTP status code was not 200.');
   } catch (e) {
+    dispatch(
+      pushNotificationMessage(
+        'error',
+        'We couldn\'t connect to the server to load processing settings from the previous session, please try refreshing the page',
+        5,
```

Comment:
The user doesn't need to know the technical background behind what caused the error to occur.

We can just say:

We couldn't load your processing settings. Please check your internet connection and try refreshing the page.